charts/cryostat/templates/cryostat_deployment.yaml

```diff
@@ -216,3 +222,8 @@ spec:
         secret:
           secretName: {{ .Release.Name }}-proxy-tls
       {{- end }}
+      {{- range .Values.core.config.eventTemplates.configMapNames}}
+      - name: {{ . }}
+        configMap:
```

Comment:
Also, should we enforce that these configmaps are available at the time of installation (i.e. `optional: false`)?

Reply:
That's a fair point.

Reply:
From what I can tell, I think `optional: false` is the default (which makes sense), but I can mark them as explicitly non-optional.

Reply:
Oh right, sounds good!